ui/src/components/workflow/executions/WorkflowList.js

```diff
@@ -231,6 +233,18 @@ const Workflow = React.createClass({
     const workflowNames = this.state.workflows?this.state.workflows:[];
     const statusList = ['RUNNING','COMPLETED','FAILED','TIMED_OUT','TERMINATED','PAUSED'];
 
+    //secondary filter to match sure we only show workflows that match the the status
+    //console.log(this.state);
+     var currentStatus = this.state.status;
```

Comment:
what if user add multiple statuses to the filter?

Reply:
@cyzhao oh, didn't realize there is a way to do it on the UI side, will recheck